source/Core/LangSupport/lang_single.cpp

```diff
@@ -0,0 +1,9 @@
+#include "Translation.h"
+
+const char *translatedString(const char *const *strs) { return strs[0]; }
+
+bool settings_setLanguageSwitch(void) { return false; }
+
+bool settings_displayLanguageSwitch(void) {
```

Comment:
I wonder if we marked this as constexpr if it would optimise better or not (curiosity, not actually sure)

Reply:
No, `constexpr` only works when defined in the same translation unit as the call site. It doesn't affect LTO, I think.